app/models/shelter.rb

```diff
@@ -16,6 +17,14 @@ def self.order_by_number_of_pets
       .order("pets_count DESC")
   end
 
+  def self.reverse_alpha
```

Comment:
Awesome job sticking to the raw SQL for this one, making these methods in the model, then using in the index action in the controller.